package.json

```diff
@@ -82,37 +82,38 @@
     "zone.js": "0.11.4"
   },
   "devDependencies": {
-    "@angular-devkit/build-angular": "13.3.7",
-    "@angular-eslint/eslint-plugin": "13.0.1",
-    "@angular-eslint/eslint-plugin-template": "13.0.1",
-    "@angular-eslint/template-parser": "13.0.1",
-    "@angular/cli": "13.3.7",
-    "@angular/compiler-cli": "13.3.11",
-    "@angular/language-service": "13.3.11",
+    "@angular-devkit/build-angular": "14.0.1",
+    "@angular-eslint/eslint-plugin": "13.2.1",
```

Comment:
Is this what NX migrates to? As I know @anglular-eslint is currently running alpha on version `14`, but if this is manual we should properly consider creating a task of keeping an eye on this future release as well.

Reply:
I don't remember whether it was an automatic migration. But will also suggest to update our deps more often so the new version will naturally come up during the next update.